src/controllers/transfer/transfer.ts

```diff
@@ -193,6 +195,15 @@ export class TransferController extends EventEmitter {
     this.emitUpdate()
   }
 
+  get shouldTrackLatestBroadcastedAccountOp() {
+    return this.#shouldTrackLatestBroadcastedAccountOp
+  }
+
+  set shouldTrackLatestBroadcastedAccountOp(value: boolean) {
+    this.#shouldTrackLatestBroadcastedAccountOp = value
+    this.emitUpdate()
+  }
```

Comment:
This is not a commonly used pattern and can result in unnecessary emit updates if someone doesn't know that there is a setter. Also, this results in an unnecessary emit update in `update()` every time it's called. Finally, an update isn't needed in `handleCleanUpOnPortDisconnect` as opening the popup again will get the entire state from the controller.